src/js/options.js

```diff
@@ -201,11 +207,35 @@ var SettingsTab = {
   initialize: function () {
     fetch("views/settings.html")
       .then((response) => response.text())
-      .then((htmlData) => {
+      .then(async (htmlData) => {
         // Insert template file.
         document.querySelector('main section[data-tab="settings"]').innerHTML =
           htmlData;
 
+        await SettingsTab.helper(
+          CONFIG_FILE_OPTIONS,
+          "template_config",
+          "config"
+        );
+        await SettingsTab.helper(
+          SEARCH_RESULT_OPTIONS,
+          "template_search-results",
+          "search-results"
+        );
+        await SettingsTab.helper(
+          MERGE_OPTIONS,
+          "template_merge-options",
+          "merge-options"
+        );
+
+        // Enable popovers.
+        $(function () {
+          $('[data-toggle="popover"]').popover({
+            trigger: "hover",
+            html: true,
+          });
+        });
```

Comment:
Does the popover's implementation from [Boostrap.native](http://thednp.github.io/bootstrap.native/#componentPopover) isn't good?

> We originally removed jQuery because: it wasn't really need; it opens the door to bad programming practices; for security (isn't that jQuery is itself insecure, but it's targeted by hackers).  
> When implementing the modal for edit the JSON settings, Boostrap.Native was added to _avoid_ having to re-introduce jQuery.  
> However if Bootstrap.Native isn't doing his job (for example: don't works correctly for popovers), then we should remove it and use only jQuery, but we shouldn't use **both** jQuery and Bootstrap.Native.

Reply:
I've used Bootstrap.native for dropdown and popover and removed jquery